src/main/java/frc/robot/Limelight.java

```diff
@@ -22,4 +62,202 @@ public void togglePipeline() {
         }
 
     }
+
+    /**
+     * Gets the horizontal offset angle from networkTable
+     * @return The horizontal offset angle from the limelight to the target
+     */
+    public Double getHorizontalOffset() {
+        
+        if(hasTargets()){
+
+        return table.getEntry("tx").getNumber(0).doubleValue();
+        
+        }
+        return Double.NaN;
+    }
+
+   
+
+    /**
+     * Gets the vertical offset angle from the limelight to the target
+     * @return The vertical angle from the limelight to the target
+     */
```

Comment:
explain what "vertical offset angle" means: vertical angle from crosshair to target?